src/stream/quic_lb/ngx_stream_quic_lb_retry_service.c

```diff
@@ -241,7 +273,7 @@ ngx_stream_quic_lb_validate_share_state_token(ngx_connection_t *c,
     aad.len = pp - plaintext_buf.data;
 
     token_body_enc.data = cp;
-    token_body_enc.len = pkt->token.len - (cp - pkt->token.data) - NGX_QUIC_RETRY_ICV_LEN;
+    token_body_enc.len = pkt->token.len - (cp - pkt->token.data);
```

Comment:
change variable name from token_body_enc  to token_body_enc_and_tag  maybe more accurate.

Also, CI/CD needed to find these problems.